frontend/src/components/user/UserManageModal.vue

```diff
@@ -18,19 +18,25 @@ const authzStore = useAuthZStore();
 
 // State
 const visible = defineModel<boolean>('visible');
-const selectableGroups: Ref<Array<GroupName>> = ref([]);
+const selectableGroups: Ref<Map<string, GroupName>> = ref(new Map());
 const group: Ref<GroupName | undefined> = ref(undefined);
 const yarsGroups: Ref<Array<Group>> = ref([]);
```

Comment:
Doesn't look like this is used outside of `onMounted`. I think we can removed the state value and just make it a `const` in `onMounted`.